src/SFA.DAS.EAS.Web/Authentication/UserInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.EmployerCommitments.Web.Authentication
+{
+    public class UserInfo

```

Comment:
Why have we had to add this? OwinWrapper.GetClaimValue("sub") has been sufficient on other methods.

Reply:
I've now removed this and used the existing. This had been added by Maulik to wrap up the OwinWrapper stuff, but he hadn't applied it elsewhere so it wasn't adding any benefit!